### PR TITLE
Configurable search providers from config

### PR DESCRIPTION
**Pull request by zoran995**
TerriaMap work for #5145

### CODE REVIEW

File: wwwroot/config.json
Comment (ljowen) [inline wwwroot/config.json:81]:
```suggestion
    "searchProviders": [
      {
        "id": "search-provider/cesium-ion-search-provider",
        "type": "cesium-ion-search-provider",
        "url": "https://api.cesium.com/v1/geocode/search",
        "flightDurationSeconds": 1.5
      },
```

**Review by ljowen — COMMENTED** [review]
Looks good! Added CesiumIon search as that will become the default

File: wwwroot/config.json
Comment (zoran995) [reply wwwroot/config.json:81]: added, just have to wait for terriajs release before merging this

**Review by nf-s — APPROVED** [review]
Thanks a lot @zoran995 !
This is an awesome improvement.
Thanks for reviewing the TerriaJS PR @ljowen !

**Comment by pjonsson** [comment]
@nf-s since this PR updated CHANGES.md and added a date that is now a few days old, should there be a release 0.0.9 of TerriaMap?

**Comment by pjonsson** [comment]
@nf-s Thank you!

**Pull request merged**
